spec/person_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe Person do
+  let(:person) { described_class.new(20, 'John Doe', parent_permission: true) }
+
+  describe '#initialize' do
+    it 'initializes a Person object' do
+      expect(person).to be_an_instance_of(Person)
+    end
+
+    it 'sets the name, age, and parent_permission' do
+      expect(person.name).to eq('John Doe')
+      expect(person.age).to eq(20)
+      expect(person.instance_variable_get(:@parent_permission)).to eq(true)
+    end
+
+    it 'generates a unique id' do
+      expect(person.id).not_to be_nil
+    end
+
+    it 'initializes an empty rentals array' do
+      expect(person.rentals).to be_empty
+    end
```

Comment:
- [ ] It would be better if we write test for our `add_rental` method, let us properly test the `add_rental` method so it behaves as we expect. For example; adding a rental involves a person and a book

Reply:
well resolved! ✔️